messages/hww.proto

```diff
@@ -264,6 +264,25 @@ message RestoreFromMnemonicRequest {
   int32 timezone_offset = 2;
 }
 
+message DisplayBase32Request {
+  bytes msg = 1;
+}
+
+
+message BaseSetConfigRequest {
```

Comment:
also here, albeit very verbose, i'd be explicit: `BitBoxBaseSetConfigRequest`.

Reply:
ack